<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
	<x>0</x>
	<y>0</y>
	<width>400</width>
	<height>428</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
	<width>300</width>
	<height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Project properties</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
	<layout class="QVBoxLayout" name="verticalLayout">
	 <item>
	  <widget class="QTabWidget" name="tabWidget">
	   <property name="currentIndex">
		<number>0</number>
	   </property>
	   <widget class="QWidget" name="local">
		<attribute name="title">
		 <string>Local File System</string>
		</attribute>
		<layout class="QGridLayout" name="gridLayout_3">
		 <property name="leftMargin">
		  <number>3</number>
		 </property>
		 <property name="topMargin">
		  <number>0</number>
		 </property>
		 <property name="rightMargin">
		  <number>3</number>
		 </property>
		 <property name="bottomMargin">
		  <number>0</number>
		 </property>
		 <property name="horizontalSpacing">
		  <number>6</number>
		 </property>
		 <item row="0" column="0">
		  <layout class="QGridLayout" name="localGridLayout">
		   <property name="spacing">
			<number>3</number>
		   </property>
		   <item row="0" column="0">
			<widget class="QLabel" name="localLabelProjectPath">
			 <property name="text">
			  <string>Project Path:</string>
			 </property>
			 <property name="alignment">
			  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
			 </property>
			</widget>
		   </item>
		   <item row="0" column="1">
			<widget class="QLineEdit" name="localEditProjectPath"/>
		   </item>
		   <item row="0" column="2">
			<widget class="QPushButton" name="localBrowse">
			 <property name="text">
			  <string>...</string>
			 </property>
			</widget>
		   </item>
		   <item row="1" column="0">
			<widget class="QLabel" name="localLabelUrl">
			 <property name="text">
			  <string>URL:</string>
			 </property>
			 <property name="alignment">
			  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
			 </property>
			</widget>
		   </item>
		   <item row="1" column="1" colspan="2">
			<widget class="QLineEdit" name="localEditURL"/>
		   </item>
		   <item row="2" column="1">
			<spacer name="localBottomVerticalSpacer">
			 <property name="orientation">
			  <enum>Qt::Vertical</enum>
			 </property>
			 <property name="sizeHint" stdset="0">
			  <size>
			   <width>20</width>
			   <height>40</height>
			  </size>
			 </property>
			</spacer>
		   </item>
		  </layout>
		 </item>
		</layout>
	   </widget>
	   <widget class="QWidget" name="rds">
		<attribute name="title">
		 <string>Adobe RDS</string>
		</attribute>
		<layout class="QGridLayout" name="gridLayout_8">
		 <item row="0" column="0">
		  <layout class="QVBoxLayout" name="RDBVerticalLayout">
		   <item>
			<widget class="QGroupBox" name="RDBRDSGroupBox">
			 <property name="title">
			  <string>RDS</string>
			 </property>
			 <layout class="QGridLayout" name="gridLayout_5">
			  <item row="0" column="0">
			   <layout class="QGridLayout" name="rdsRDSGridLayout">
				<item row="0" column="0">
				 <widget class="QLabel" name="RDS_Host_label">
				  <property name="text">
				   <string>Host:</string>
				  </property>
				  <property name="alignment">
				   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
				  </property>
				 </widget>
				</item>
				<item row="0" column="1">
				 <widget class="QLineEdit" name="RDS_Host_lineEdit"/>
				</item>
				<item row="0" column="2">
				 <widget class="QSpinBox" name="RDS_Host_spinBox">
				  <property name="minimum">
				   <number>1</number>
				  </property>
				  <property name="maximum">
				   <number>65535</number>
				  </property>
				  <property name="value">
				   <number>80</number>
				  </property>
				 </widget>
				</item>
				<item row="1" column="0">
				 <widget class="QLabel" name="RDS_Username_label">
				  <property name="text">
				   <string>Username:</string>
				  </property>
				  <property name="alignment">
				   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
				  </property>
				 </widget>
				</item>
				<item row="1" column="1" colspan="2">
				 <widget class="QLineEdit" name="RDS_Username_lineEdit"/>
				</item>
				<item row="2" column="0">
				 <widget class="QLabel" name="RDS_Password_label">
				  <property name="text">
				   <string>Password:</string>
				  </property>
				  <property name="alignment">
				   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
				  </property>
				 </widget>
				</item>
				<item row="2" column="1" colspan="2">
				 <widget class="QLineEdit" name="RDS_Password_lineEdit">
				  <property name="echoMode">
				   <enum>QLineEdit::Password</enum>
				  </property>
				 </widget>
				</item>
				<item row="3" column="1" colspan="2">
				 <widget class="QPushButton" name="RDS_Test_pushButton">
				  <property name="text">
				   <string>&amp;Test</string>
				  </property>
				 </widget>
				</item>
			   </layout>
			  </item>
			 </layout>
			</widget>
		   </item>
		   <item>
			<widget class="QGroupBox" name="RDSMappingGroupBox">
			 <property name="title">
			  <string>Mapping</string>
			 </property>
			 <layout class="QGridLayout" name="gridLayout_7">
			  <item row="0" column="0">
			   <layout class="QGridLayout" name="rdsMappingGridLayout">
				<item row="0" column="0">
				 <widget class="QLabel" name="rdsMappingLabelProjectPath">
				  <property name="text">
				   <string>Project Path:</string>
				  </property>
				  <property name="alignment">
				   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
				  </property>
				 </widget>
				</item>
				<item row="0" column="1">
				 <widget class="QLineEdit" name="lineEdit_4"/>
				</item>
				<item row="0" column="2">
				 <widget class="QPushButton" name="rdsMappingPushButtonBrowse">
				  <property name="text">
				   <string>...</string>
				  </property>
				  <property name="autoRepeatDelay">
				   <number>300</number>
				  </property>
				 </widget>
				</item>
				<item row="1" column="0">
				 <widget class="QLabel" name="rdsMappingLabelURL">
				  <property name="text">
				   <string>URL:</string>
				  </property>
				  <property name="alignment">
				   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
				  </property>
				 </widget>
				</item>
				<item row="1" column="1" colspan="2">
				 <widget class="QLineEdit" name="lineEdit_5"/>
				</item>
			   </layout>
			  </item>
			 </layout>
			</widget>
		   </item>
		   <item>
			<spacer name="RDSBottomVerticalSpacer">
			 <property name="orientation">
			  <enum>Qt::Vertical</enum>
			 </property>
			 <property name="sizeHint" stdset="0">
			  <size>
			   <width>20</width>
			   <height>40</height>
			  </size>
			 </property>
			</spacer>
		   </item>
		  </layout>
		 </item>
		</layout>
	   </widget>
	   <widget class="QWidget" name="jdwp">
		<attribute name="title">
		 <string>Sun JDWP</string>
		</attribute>
	   </widget>
	   <widget class="QWidget" name="ftp">
		<attribute name="title">
		 <string>FTP</string>
		</attribute>
	   </widget>
	   <widget class="QWidget" name="sftp">
		<attribute name="title">
		 <string>SFTP</string>
		</attribute>
	   </widget>
	  </widget>
	 </item>
	 <item>
	  <widget class="QDialogButtonBox" name="buttonBox">
	   <property name="orientation">
		<enum>Qt::Horizontal</enum>
	   </property>
	   <property name="standardButtons">
		<set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
	   </property>
	  </widget>
	 </item>
	</layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
	<hint type="sourcelabel">
	 <x>248</x>
	 <y>254</y>
	</hint>
	<hint type="destinationlabel">
	 <x>157</x>
	 <y>274</y>
	</hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
	<hint type="sourcelabel">
	 <x>316</x>
	 <y>260</y>
	</hint>
	<hint type="destinationlabel">
	 <x>286</x>
	 <y>274</y>
	</hint>
   </hints>
  </connection>
 </connections>
</ui>
